pkg/api/const.go

```diff
@@ -397,7 +398,7 @@ const (
 	// DefaultKubernetesCtrlMgrUseSvcAccountCreds is "true", see --use-service-account-credentials at https://kubernetes.io/docs/admin/kube-controller-manager/
 	DefaultKubernetesCtrlMgrUseSvcAccountCreds = "false"
 	// DefaultKubernetesCloudProviderBackoff is false to disable cloudprovider backoff implementation for API calls
-	DefaultKubernetesCloudProviderBackoff = true
+	DefaultKubernetesCloudProviderBackoff = false
```

Comment:
@khenidak is it correct that we should change the current default behavior so that < 1.15 clusters have backoff disabled by default? It's my understanding that you can't use v2 unless you're on >= k8s 1.15. What about < k8s 1.15? Should those clusters get the pre-v2 backoff enabled or disabled by default?